app/src/main/kotlin/at/bitfire/davdroid/ui/AccountsModel.kt

```diff
@@ -203,6 +204,12 @@ class AccountsModel @AssistedInject constructor(
             }
         }
 
+    /** whether the calendar provider is missing or disabled */
+    val calendarProviderInaccessible = !appInstalledAndEnabled("com.android.providers.calendar")
```

Comment:
We could put in into a `packageChangedFlow` (see `BroadcastReceiverFlow.kt`). Then it would even work dynamically when you switch to Settings, disable the Calendar provider and switch back to DAVx⁵.

Reply:
I was wondering how to make that work. I did not know such changes would be broadcasted and could be received with a flow. That is really cool. I will try to look into it!